framework/src/modules/token/method.ts

```diff
@@ -753,7 +753,7 @@ export class TokenMethod extends BaseMethod {
 			throw new Error('All tokens are supported.');
 		}
 
-		if (tokenID.equals(this.getMainchainTokenID()) || chainID.equals(this._config.ownChainID)) {
+		if (tokenID.equals(this.getTokenIDLSK()) || chainID.equals(this._config.ownChainID)) {
 			throw new Error('Cannot remove support for the specified token.');
 		}
 
```

Comment:
Reminder: In [this PR](https://github.com/LiskHQ/lisk-sdk/pull/8806/), the `beforeCrossChainMessageForwarding` function of token module (file token/cc_method.ts) is modified and an extra check involving `getMainChainTokenID()` is added. 

Not sure what is the best way to handle this, but I am leaving a comment in both PRs so that this is not overlooked and the overall code is consistent.

Reply:
in this PR, it's just changing the name, so it should be handled in the other PR which changes forwarding, or later. anyway it should fail on the build

Reply:
Sounds good, I just wanted to avoid overlooking this.